### PR TITLE
responsive layout of book details was added

### PR DESCRIPTION
**Pull request by shernanm**
* Responsive layout of book details was added.

## Screenshot
image

## Trello Card
* [Trello Card](https://trello.com/c/F8PnopXK)

### CODE REVIEW

File: scss/detail.scss [hunk 1]
```diff
@@ -9,21 +13,25 @@
   font-size: 25px;
   font-weight: bold;
   margin-right: 5px;
+  position: relative;
 
   &::before {
     background: $lemon;
-    content: "";
+    content: '';
```
Comment (CarlosHWolox) [inline scss/detail.scss]: why this change?

File: scss/detail.scss [hunk 2]
```diff
@@ -39,59 +47,87 @@
 .container {
   box-shadow: 0 3px 6px rgba(0, 0, 0, 0.16), 0 3px 6px rgba(0, 0, 0, 0.23);
   display: flex;
-  margin: 20px auto;
+  margin: 80px auto;
   padding: 20px;
   position: relative;
-  width: 650px;
+  width: 670px;
+  @media screen and (max-width: 1024px) {
```
Comment (CarlosHWolox) [inline scss/detail.scss]: We use the media screen at the end of the file and target the classes inside of them

**Review by CarlosHWolox — CHANGES_REQUESTED** [review]

File: scss/detail.scss (hunk 1 above)
Comment (shernanm) [reply scss/detail.scss]: A tool that I used and modified the quotes.

File: bookDetails.html [hunk 3]
```diff
@@ -10,8 +10,8 @@
   </head>
 
   <body>
-    <a class="back-to-index" href="./index.html">Atrás</a>
     <div class="container">
+      <a class="back-to-index" href="./index.html">Atrás</a>
       <div class="container-img-book">
```
Comment (CarlosHWolox) [inline bookDetails.html]: Is this necessary?

**Review by CarlosHWolox — CHANGES_REQUESTED** [review]

**Review by CarlosHWolox — APPROVED** [review]
